packages/perseus-editor/src/__stories__/editor-page-with-storybook-preview.tsx

```diff
@@ -61,9 +67,10 @@ function EditorPageWithStorybookPreview(props: Props) {
                 answerArea={answerArea}
                 question={question}
                 hints={hints}
-                frameSource="about:blank"
-                previewURL="about:blank"
-                itemId="1"
+                // These urls lead to the PerseusFrameComponent story
+                frameSource="/iframe.html?id=perseuseditor-perseus-frame--frame&viewMode=story"
+                previewURL="/iframe.html?id=perseuseditor-perseus-frame--frame&viewMode=story"
```

Comment:
Does this leverage a Storybook iframe feature?  I'd be mildly concerned that this could break if someone re-names a story.  We could also just put a container page in the [Vite static asset](https://storybook.js.org/docs/configure/integration/images-and-assets#serving-static-files-via-storybook-configuration) directory if you think that's more resilient to changes.

Reply:
I didn't need to leverage much of storybook at all! I tried to avoid too much storybook middleware as it seemed to diverge a little from how we would set it up in production. 

This currently links to the Perseus Frame Component's story in "story" mode (AKA the full page view). I think the vite static asset directory could make a lot of sense, as it's definitely possible for this to break if someone moves the Perseus Frame component story. While I don't think it's likely that someone will rename the story, I think the static asset directory is a good way to keep us safe. :) I'll look into getting that working!